vcl_templates/www.vcl.erb

```diff
@@ -40,6 +40,55 @@ backend F_origin {
 <% end -%>
 }
 
+<%- if config['replatforming_traffic_split'] %>
+backend F_eks_origin {
+    .connect_timeout = 5s;
+    .dynamic = true;
+    .port = "443";
+    .host = "www-origin.eks.staging.govuk.digital";
```

Comment:
template things out now.

Reply:
I didn't want to template because it's only going to be used for this experiment. Once I've verified it works in staging, this will be rolled back and I'll put in a new PR with the actual production values.

Reply:
I'm ok with either approach (as long as you revalidate the solution in staging if you change the approach). If we want to change the approach, it'd be good to be specific about the practical reasons behind that as applicable to our use case.